### ISSUE TITLE
Add command/keybinding for focusing the files tab in the Changes view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#308327]
Currently the Changes view in the Agents window has a `Focus Changes View` command (`workbench.action.agentSessions.focusChangesView`, `Cmd+Shift+H`) that opens the pane composite and focuses the view. However, there is no dedicated command or keybinding for focusing specifically the **files tab/tree** within the Changes view.

This should be aligned with how the regular workbench has `workbench.explorer.fileView.focus` for focusing the file explorer tree.

### Proposal

- Add a `focusCommand` to the Changes view descriptor (similar to `workbench.explorer.fileView.focus` on the explorer view descriptor)
- This gives a dedicated command that can be keybound for focusing the files tree within the Changes view
